app/models/ems_event.rb

```diff
@@ -176,7 +176,7 @@ def self.first_chained_event(ems_id, chain_id)
   def parse_event_metadata
     data = full_data || {}
     [
-      event_type == "datawarehouse_alert" ? message : nil,
+      event_type == "monitoring_alert" ? message : nil,
```

Comment:
Overloading event_type then switching it to message feels pretty hacky, this means the event_type won't be persisted to the EventStreams table which makes it harder to diagnose from a database.

I know that is how it is currently done but there has to be a better way